axelar/assetlist.json

```diff
@@ -410,6 +410,37 @@
 	      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/moonbeam/images/glmr.png"
       },
       "coingecko_id": "wrapped-moonbeam"
+    },
+    {
+      "description": "Wrapped Matic on Axelar",
+      "denom_units": [
+        {
+          "denom": "wmatic-wei",
+          "exponent": 0
+        },
+        {
+          "denom": "wmatic",
+          "exponent": 18
+        }
+      ],
+      "base": "wmatic-wei",
+      "name": "Wrapped Matic",
+      "display": "wmatic",
+      "symbol": "WMATIC",
+      "traces": [
+        {
+          "type": "bridge",
+          "platform": "polygon",
+          "non-cosmos": true,
+          "base_denom": "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270",
+          "provider": "Axelar"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/polygon/images/wmatic.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/polygon/images/wmatic.svg"
+      },
+      "coingecko_id": "wmatic"
```

Comment:
We should probably just use the Matic coingecko id, not wmatic

https://www.coingecko.com/en/coins/polygon